.github/workflows/trigger-nightly.yml

```diff
@@ -0,0 +1,46 @@
+name: Trigger nightly build
+
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '0 * * * *'
```

Comment:
As mentioned in the meeting - we should change this to weekdays only.

Reply:
Done.